File: film-information-protal-web/src/main/webapp/js/service/filmDescService.js
//服务层
app.service('filmDescService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../filmDesc/findAll.do');
	}
	//分页 
	this.findPage=function(page,rows){
		return $http.get('../filmDesc/findPage.do?page='+page+'&rows='+rows);
	}
	//查询实体
	this.findOne=function(id){
		return $http.get('../filmDesc/findOne.do?id='+id);
	}
	//修改 
	this.update=function(entity){
		return  $http.post('../filmDesc/update.do',entity );
	}
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('../filmDesc/search.do?page='+page+"&rows="+rows, searchEntity);
	}    	
});

File: film-information-service-interface/src/main/java/com/michael/service/FilmDescService.java
package com.michael.service;

import com.michael.pojo.Film;
import com.michael.pojo.FilmDesc;
import entity.PageResult;

import java.util.List;

/**
 * 服务层接口
 * @author Administrator
 *
 */
public interface FilmDescService {

	/**
	 * 返回全部列表
	 * @return
	 */
	public List<FilmDesc> findAll();
	
	
	/**
	 * 返回分页列表
	 * @return
	 */
	public PageResult findPage(int pageNum, int pageSize);
	
	

	/**
	 * 修改
	 */
	public void update(FilmDesc film);
	

	/**
	 * 根据ID获取实体
	 * @param id
	 * @return
	 */
	public FilmDesc findOne(Long id);
	
	

	/**
	 * 分页
	 * @param pageNum 当前页 码
	 * @param pageSize 每页记录数
	 * @return
	 */
	public PageResult findPage(FilmDesc film, int pageNum, int pageSize);
	
}
